satpy/readers/seviri_base.py

```diff
@@ -353,6 +353,62 @@
                          'ALPHA': 0.9981,
                          'BETA': 0.5635}}
 
+# Calibration coefficients from Meirink, J.F., R.A. Roebeling and P. Stammes, 2013:
+# Inter-calibration of polar imager solar channels using SEVIRI, Atm. Meas. Tech., 6,
+# 2495-2508, doi:10.5194/amt-6-2495-2013
+
+# To obtain the slope for the calibration, one should use the routine get_seviri_meirink_slope
+
+# Epoch for the MEIRINK re-calibration
+DATE_2000 = datetime(2000, 1, 1)
+
+MEIRINK_COEFS = {}
+
+# Meteosat-8
+
+MEIRINK_COEFS[321] = {'VIS006': (24.346, 0.3739),
```

Comment:
BTW, what do you think of storing slope and offset in a dictionary `{"slope": 12.345, "offset": 0.123}`? I think that would improve readability.

Reply:
Oh, and can you please add a comment specifying the physical units of these coefficients?

Reply:
> BTW, what do you think of storing slope and offset in a dictionary `{"slope": 12.345, "offset": 0.123}`? I think that would improve readability.

That would look like:
```.py
gain_offset_dict = self.get_gain_offset()
gain = gain_offset_dict['gain']
offset = gain_offset_dict['offset']
```
is it worth it?

Reply:
I'd say yes :cat: But I don't have a strong opinion, so feel free to leave it as it is.